test/modules/Relayer/Program.cs

```diff
@@ -121,6 +131,20 @@ static async Task<MessageResponse> ProcessAndSendMessageAsync(Message message, o
                     await ModuleUtil.ReportTestResultAsync(testResultReportingClient, Logger, testResultSent);
                     Logger.LogInformation($"Successfully sent message: trackingid={trackingId}, batchId={batchId}, sequenceNumber={sequenceNumber}");
                 }
+                else
+                {
+                    int uniqueResultsExpected = int.Parse(
+                        Settings.Current.UniqueResultsExpected.Expect<ArgumentException>(() => throw new ArgumentException("Must supply this value if in ReceiveOnly mode")));
```

Comment:
I don't think you need to be explicit about the template argument to `Expect`. The compiler should be able to figure it out from your closure.

Reply:
No, if it's complaining, might as well use the exact type.